packages/base-styles/_variables.scss

```diff
@@ -76,6 +76,8 @@ $block-selected-to-content: $block-edge-to-content - $block-selected-margin - $b
 $block-selected-child-to-content: $block-selected-to-content - $block-selected-child-margin - $block-selected-child-border-width;
 $block-custom-appender-to-content: $block-selected-margin - $block-selected-border-width;
 $block-media-container-to-content: $block-selected-child-margin + $block-selected-border-width;
+$block-selected-vertical-margin-descendant: 2 * $block-selected-to-content;
+$block-selected-vertical-margin-child: $block-edge-to-content;
```

Comment:
When block is a descendant each block gets bottom margin of double of `$block-selected-to-content`. Then it will equals the original space between blocks (it also includes padding applied by our logic when parent is selected).

Space between child is set to `$block-edge-to-content` because our bordering rule apply also some margin to each block (with `$block-selected-margin` and `$block-selected-border-width`)

So in each case space between block content should be equal.

Reply:
Ah, after testing this for a bit I think I understand why this works that way. The double space is so that the children stay in place after being selected:

![Untitled 2](https://user-images.githubusercontent.com/8739/73646737-59840480-467a-11ea-9090-6a6a00aa82b3.png)

Reply:
Yeah, with this there is less "jumpy-effect" when selection change.

I think after we implement ideal version of FloatingToolbar we can also reduce it even more.

You think above implementation make sense ?

Reply:
Yeah, once I figured out what it was doing it made sense. It wasn't super clear just by looking at the variable names, but I don't have a better suggestion

Reply:
ok, thanks !